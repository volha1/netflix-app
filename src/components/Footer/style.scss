@import '../../common/scss/base.scss';

.footer {
  background-color: $gray;
  padding: 25px 0px;

  .logo {
    text-align: center;
  }
}
