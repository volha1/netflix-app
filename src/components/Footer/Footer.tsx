import React, { ReactElement } from 'react';
import './style.scss';

const Footer = (): ReactElement => {
  return (
    <div className="footer">
      <div className="content">
        <p className="logo">
          <span>netflix</span>roulette
        </p>
      </div>
    </div>
  );
};

export default Footer;
