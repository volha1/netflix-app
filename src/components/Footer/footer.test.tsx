import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './Footer';

it('Footer snapshot', () => {
  const component = renderer.create(<Footer />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
