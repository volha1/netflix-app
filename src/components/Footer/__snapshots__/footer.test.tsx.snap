// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<div
  className="footer"
>
  <div
    className="content"
  >
    <p
      className="logo"
    >
      <span>
        netflix
      </span>
      roulette
    </p>
  </div>
</div>
`;
